packages/eas-json/src/accessor.ts

```diff
@@ -90,7 +96,7 @@ export class EasJsonAccessor {
       this.isJson5 = true;
       return rawJSON;
     } catch (originalError: any) {
-      const err = new Error('Found invalid JSON in eas.json.');
+      const err = new InvalidEasJsonError(`Found invalid character in ${chalk.bold('eas.json')}.`);
```

Comment:
Are you sure this message will always make sense? e.g. what happens if a file is empty?

Reply:
<img width="347" alt="Screenshot 2022-09-29 125153" src="https://user-images.githubusercontent.com/719641/193012947-e24959a2-5c5c-43ce-a173-dd8979ec0767.png">

Maybe we can separate this edge case, I will look at it.

Reply:
Finally I have changed the logic a bit and reverted the original error message, when there is no information about problem location in the file.